lib/convection/model/template/resource/aws_auto_scaling_auto_scaling_group.rb

```diff
@@ -30,6 +30,51 @@ class AutoScalingGroup < Resource
           def render(*args)
             super.tap do |resource|
               render_tags(resource)
+              @update_policy.render(resource) unless @update_policy.nil?
+            end
+          end
+
+          def update_policy(&block)
+            @update_policy = UpdatePolicy.new
+            @update_policy.instance_exec(&block)
+          end
+
+          ##
+          # UpdatePolicy for AWS::AutoScaling::AutoScalingGroup
+          ##
+          class UpdatePolicy
```

Comment:
The top-level resource.rb file currently handles the other supported resource attributes (DependsOn, DeletionPolicy).  I'll try to separate out UpdatePolicy; I'm loathe to change the other attributes if there are no tests in place.